phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/application/skin/ApplicationInformationPanelSkin.java

```diff
@@ -198,7 +202,29 @@ private void updateScripts(final GridPane scriptGrid) {
                 }
             });
 
-            scriptGrid.addRow(i, scriptName, installButton);
+            OperatingSystem curOs = new OperatingSystemFetcher().fetchCurrentOperationSystem();
+            Label lTesting = new Label();
+            if (script.getTestingOperatingSystems().contains(curOs)) {
+                assignIcon(lTesting, "Testing", "testingIcon", 30);
+            }
+            Label lCommercial = new Label();
+            if (!script.isFree()) {
+                assignIcon(lCommercial, "Commercial", "commercialIcon", 30);
+            }
+            Label lPatch = new Label();
+            if (script.isRequiresPatch()) {
+                assignIcon(lPatch, "Patch required", "patchIcon", 30);
+            }
+            Label lOs = new Label();
+            if (!script.getCompatibleOperatingSystems().contains(curOs)) {
+                assignIcon(lOs, "All Operating Systems", "osIcon", 30);
+            }
+            Label lSpace = new Label();
+            lSpace.setPrefSize(30, 30);
+
+            javafx.scene.layout.HBox iconBox = new javafx.scene.layout.HBox(lTesting, lCommercial, lPatch, lOs, lSpace);
```

Comment:
I don't think we need the fully qualified Class name, which includes the package name, here, i.e. `javafx.scene.layout.HBox`